src/compiler/transformers/map-imports-to-path-aliases.ts

```diff
@@ -1,5 +1,5 @@
-import { normalizePath } from '@utils';
-import { dirname, relative } from 'path';
+import { normalizePath, relative } from '@utils';
```

Comment:
Here I think we do. The wrapped functions call `normalizePath` with `false` for the second positional argument which disables automatic insertion of `./`. They do that so that while they get normalized from POSIX -> POSIX and Windows -> POSIX they will otherwise keep the same output that you'd get from `join` and `resolve`, i.e. `path.join("foo", "bar")` returns `"foo/bar"` not `"./foo/bar"`.

Since we already had a `normalizePath` call here and since that argument defaults to `true` the existing behavior when declaring `importPath` here (i.e. what happens currently on `main`) would result in adding `"./"` as a prefix, so we need to keep the wrapping `normalizePath` call here to ensure that keeps happening.